data/mods/Sorcerer/generated_code/spell_data_dump.txt

```diff
@@ -0,0 +1,2116 @@
+[
+  {
+    "id": "banishment_lesser",
```

Comment:
Is this a necessary bit, or is it just a temp file that can be removed later

Reply:
It is not functional. It could help people with changing tools/spell_data.txt.
But if they need the data dump then they could just re-generate it themselves. I guess I'll just remove the file.